app/api/routes.py

```diff
@@ -1757,7 +1757,7 @@
 api.route(
     VideoStreamModeratorDetail,
     'video_stream_moderator_detail',
-    '/video-stream-moderators/<int:id>',
+    '/video-stream-moderators/<int:video_stream_id>',
```

Comment:
This should be moderator_id

Reply:
In fact it should be just ID

Reply:
it's failing tests when I am using `/video-stream-moderators/<int:id>`